ci-operator/config/openshift/microshift/openshift-microshift-main__periodics_rhel-8-x86_64.yaml

```diff
@@ -0,0 +1,159 @@
+    COPY validate-microshift /tmp/validate-microshift
+  from: libvirt-installer
+  inputs:
+    src:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/microshift/validate-microshift
+  to: microshift-validate
+- dockerfile_literal: |
+    FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.19-openshift-4.13 as builder
+    RUN dnf install --setopt=tsflags=nodocs -y \
+        rpm-build glibc-static gcc selinux-policy-devel util-linux && \
+        dnf clean all && \
+        rm -rf /var/cache/dnf/*
+    RUN ls -lA
+    RUN make rpm
+    FROM libvirt-installer
+    USER root
+    ENV HOME=/tmp/secret
+    RUN dnf install -y jq && dnf clean all && rm -rf /var/cache/dnf/*
+    ENV PAYLOAD_PATH=$HOME/payload
+    WORKDIR $PAYLOAD_PATH
+    COPY oc openshift-tests usr/bin/
+    RUN chmod 755 usr/bin/oc && ln -rs usr/bin/oc usr/bin/kubectl
+    COPY --from=builder /go/src/github.com/openshift/microshift/scripts/devenv-builder/configure-vm.sh usr/bin/configure-vm.sh
+    RUN chmod 755 usr/bin/configure-vm.sh
+    COPY --from=builder /go/src/github.com/openshift/microshift/_output/rpmbuild/RPMS/*/*.rpm packages/
+    RUN tar -cvf payload.tar . && chmod -R a+rw "${HOME}"
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    libvirt-installer:
+      as:
+      - libvirt-installer
+    src:
+      as:
+      - registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.19-openshift-4.13
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/openshift-tests
+  to: dependency-payload
+- dockerfile_literal: |
+    # build
+    FROM ubi-minimal
+    ENV YQ_URL=https://github.com/mikefarah/yq/releases/download/v4.26.1/yq_linux_amd64
+    ENV YQ_HASH=9e35b817e7cdc358c1fcd8498f3872db169c3303b61645cc1faf972990f37582
+    RUN echo -n "${YQ_HASH} -" > /tmp/sum.txt ; \
+        if ! (curl -Ls "${YQ_URL}" | tee /tmp/yq | sha256sum -c /tmp/sum.txt &>/dev/null); then \
+          echo "ERROR: Expected file at ${YQ_URL} to have checksum ${YQ_HASH} but instead got $(sha256sum </tmp/yq | cut -d' ' -f1)" ; \
+          exit 1 ; \
+        fi ; \
+        chmod +x /tmp/yq && mv /tmp/yq /usr/bin/yq
+  from: ubi-minimal
+  to: yq-cli
+- dockerfile_literal: |
+    FROM src
+    COPY yq /usr/bin/yq
+    RUN yum install -y --nodocs python3-devel && \
+        pip3 install pygithub GitPython && \
+        dnf clean all && \
+        rm -rf /var/cache/dnf/*
+  from: src
+  inputs:
+    yq-cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/yq
+  to: microshift-rebase
+releases:
+  arm64-latest:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
+      version: "4.13"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 800Mi
```

Comment:
What do you think about keeping all versions/arches for single branch in one file?
Instead of
```
openshift-microshift-main__periodics_rhel-8-x86_64.yaml
openshift-microshift-main__periodics_rhel-9-aarch64.yaml
openshift-microshift-main__periodics_rhel-9-x86_64.yaml
```
just
```
openshift-microshift-main__periodics.yaml
```

There's ~120 lines that are the same and only ~40 that are different. 
Right now we duplicate `test-unit`, `test-srpm`, and even `rebase-on-nightlies` - all three are arch insensitive in a sense.
(For future I'd go further and put periodics and presubmits in one file, but this is out of scope for this PR)